setup.py

```diff
@@ -43,6 +43,7 @@
     install_requires=(
         "google-api-core[grpc] >= 1.22.2, < 2.0.0dev",
         "google-cloud-storage >= 1.32.0, < 2.0.0dev",
+        "google-cloud-bigquery >= 2.3.1, < 3.0.0dev",
```

Comment:
Any reason that an older version won't work? There actually aren't all that many differences between 1.x and 2.x in `google-cloud-bigquery`, just some slightly different types for the BQML query stats.

Reply:
Dropped lower bound to `1.15.0` - [the earliest ref docs I see with `list_rows()`](https://googleapis.dev/python/bigquery/1.15.0/generated/google.cloud.bigquery.client.Client.html#google.cloud.bigquery.client.Client.list_rows)